internal/services/fluidrelay/fluid_relay_servers_resource_test.go

```diff
@@ -405,6 +427,50 @@ resource "azurerm_fluid_relay_server" "test" {
 `, f.templateWithCMK(data), data.RandomInteger, data.Locations.Primary)
 }
 
+func (f FluidRelayResource) customerManagedKeyVersionless(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+
+
+%[1]s
+
+resource "azurerm_fluid_relay_server" "test" {
+  name                = "acctestRG-fuildRelayServer-%[2]d"
+  resource_group_name = azurerm_resource_group.test.name
+  location            = azurerm_resource_group.test.location
+  identity {
+    type         = "UserAssigned"
+    identity_ids = [azurerm_user_assigned_identity.test.id]
+  }
+  customer_managed_key {
+    key_vault_key_id          = azurerm_key_vault_key.test.versionless_id
+    user_assigned_identity_id = azurerm_user_assigned_identity.test.id
+  }
+}
+`, f.templateWithCMK(data), data.RandomInteger, data.Locations.Primary)
+}
+
+func (f FluidRelayResource) customerManagedKeyVersionlessUpdated(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+
+
+%[1]s
+
+resource "azurerm_fluid_relay_server" "test" {
+  name                = "acctestRG-fuildRelayServer-%[2]d"
+  resource_group_name = azurerm_resource_group.test.name
+  location            = azurerm_resource_group.test.location
+  identity {
+    type         = "UserAssigned"
+    identity_ids = [azurerm_user_assigned_identity.test.id]
+  }
+  customer_managed_key {
+    key_vault_key_id          = azurerm_key_vault_key.test.versionless_id
+    user_assigned_identity_id = azurerm_user_assigned_identity.test.id
+  }
+}
+`, f.templateWithCMK(data), data.RandomInteger, data.Locations.Primary)
+}
```

Comment:
I'm having trouble spotting the difference between these two test configs

Reply:
🙈

Reply:
fixed, and re-run:

<img width="625" alt="image" src="https://github.com/user-attachments/assets/dbcecefa-aeb6-4744-aa1d-f0f40572c52a" />